evhtp.c

```diff
@@ -2680,6 +2744,10 @@ htp__ssl_add_scache_ent_(evhtp_ssl_t * ssl, evhtp_ssl_sess_t * sess)
     int                  slen;
 
     connection = (evhtp_connection_t *)SSL_get_app_data(ssl);
+    if (connection->htp == NULL)
```

Comment:
Should evhtp_unlikely be used here given that this condition is unlikely to occur?

Reply:
I'm not even sure if that code is even used, to be honest :)